ui/ts/pages/nodes.ts

```diff
@@ -309,8 +311,11 @@ module AdminViews {
           primaryContent = ctrl.RenderPrimaryStats();
         }
 
+        let nodeStatus: NodeStatus = nodeStatuses.GetStatus(ctrl.GetNodeId());
+        let updated: number = (nodeStatus ? nodeStatus.updated_at : 0);
```

Comment:
I considered doing that but decided to put the "null value handling" logic inside `Topbar` instead, since the same problem would occur if `Topbar` is used again elsewhere and was caused by a tricky edge case that only appeared with a recently started cluster server. I was about to commit and PR when I caught it :)

However, I agree that being explicit about it up front is simpler. If anyone feels strongly about it I'll be happy to change and update the PR.